generators/swift/codegen/src/ast/Func.ts

```diff
@@ -0,0 +1,63 @@
+import { AstNode, Writer } from "@fern-api/generator-commons";
+import Swift, { AccessLevel, FunctionModifier, Param } from "../swift";
+
+export declare namespace Func {
+    interface Args {
+        accessLevel?: AccessLevel,
+        modifier?: FunctionModifier,
+        name: string,
+        params?: Param[];
+        async?: "async";
+        throws?: "throws";
+        returnObject?: string;
```

Comment:
Same question as above, if you have a function that returns a class from a different module, you'll want to be able to represent that in the AST.